dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportController.java

```diff
@@ -134,10 +134,12 @@ PagingWrapper<ObjectNode> getTrackedEntities( TrackedEntityCriteria criteria,
                 ? (long) trackedEntityService.getTrackedEntityCount( queryParams, true, true )
                 : null;
 
+            int pageCount = (int) Math.ceil( count / (double) queryParams.getPageSizeWithDefault() );
```

Comment:
It might be useful to let the `PagingWrapper` do that for us. Everywhere we use the wrapper we would want the `pageCount` to be `count/pageSize` right?

Reply:
done.